lib/mayaUsd/ufe/ProxyShapeHierarchy.cpp

```diff
@@ -158,10 +158,18 @@ Ufe::AppendedChild ProxyShapeHierarchy::appendChild(const Ufe::SceneItem::Ptr& c
 #endif
 
 #ifdef UFE_V2_FEATURES_AVAILABLE
+
+#if UFE_PREVIEW_VERSION_NUM >= 2021
```

Comment:
Nit-pick: I would have ifdef'ed only the first line, but it doesn't really matter.